opa/Dockerfile

```
@@ -71,10 +73,10 @@ COPY opa/licenses /licenses
 USER stackable
 WORKDIR /stackable/opa
 
-RUN curl --fail -L https://repo.stackable.tech/repository/packages/opa/opa_linux_amd64_static_${PRODUCT} \
-     -o opa_linux_amd64_static_${PRODUCT} && \
-    chmod +x opa_linux_amd64_static_${PRODUCT} && \
-    ln -s opa_linux_amd64_static_${PRODUCT} opa
+RUN curl --fail -L "https://repo.stackable.tech/repository/packages/opa/opa_${TARGETOS}}_${TARGETARCH}_static_${PRODUCT}" \
```

* There is no README.txt in Nexus. The readme file should contain information about the download source and license, see e.g. https://repo.stackable.tech/repository/packages/hbase-operator-tools/README.txt.
(The readme file should have been created by the initial uploader.)
* The upstream file does not contain the product version, perhaps it is better to reorganize the directory structure to `/packages/opa/<version>/opa_<os>_<arch>_static`. But this would break building from prior commits.
* ```suggestion
  RUN curl --fail -L 
  "https://repo.stackable.tech/repository/packages/opa/opa_${TARGETOS}_${TARGETARCH}_static_${PRODUCT}" \
  ```